### PR TITLE
Languages: Clarify English is English (US)

### PR DESCRIPTION
**Pull request by GaryJones**
There's no global default variant of English. Those who care about having the right locale benefit from seeing the label as specifying that a particular entry means US English.

### CODE REVIEW

**Comment by tfrommen** [comment]
Hey Gary,

thanks for the PR.
I'm not yet completely sure this is right (or at least: enough), though.

It is true that both the locale and HTTP code indicate **US**  English, so we should adapt the name accordingly. But is it really "English (US)", or rather "English (United States)" just like WordPress has it?

Also, there **is** some global variant, called "[International English](https://en.wikipedia.org/wiki/International_English)" (abbreviated "English (Int)"), no? Maybe we should add it (which means it would get `en` as array key, and the US entry would get `en-us`. I remember a few related support topics over the last couple years.

Any opinion on this, @toscho?

**Comment by GaryJones** [comment]
I'd be happy with `English (United States)` (`US` is unnecessarily abbreviated).

**Comment by tfrommen** [comment]
Hey @GaryJones,

anything to add to my latest commit?

**Comment by GaryJones** [comment]
@tfrommen All looks good to me.

**Comment by tfrommen** [comment]
In plain English: thanks for making English US English, so we could add in an even more English English for English but not US English websites. :)
